spec/features/08_user_edits_review_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+feature 'visitors can edit reviews' do
+  let! (:npr) { Provider.create(name: "NPR") }
+  let! (:fresh_air) { Podcast.create(name: "Fresh Air", provider: npr) }
+  let! (:user) do
+    User.create(
```

Comment:
I would really like to see you using Factory Girl, in place of creating the objects "by hand," in a future PR.